### PR TITLE
chore: updates samples

### PR DESCRIPTION
**Pull request by thiagotnunes**
Syncs the samples of this repository with the samples in the https://github.com/GoogleCloudPlatform/java-docs-samples repository.

### CODE REVIEW

**Review by olavloite — APPROVED** [review]

**Review by skuruppu — APPROVED** [review]

File: samples/snippets/src/main/java/com/example/spanner/AddNumericColumnSample.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google Inc.
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/spanner/AddNumericColumnSample.java:2]:
```suggestion
 * Copyright 2020 Google LLC
```

File: samples/snippets/src/main/java/com/example/spanner/QueryWithNumericParameterSample.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google Inc.
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/spanner/QueryWithNumericParameterSample.java:2]:
```suggestion
 * Copyright 2020 Google LLC
```

File: samples/snippets/src/main/java/com/example/spanner/UpdateNumericDataSample.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google Inc.
```
Comment (chingor13) [inline samples/snippets/src/main/java/com/example/spanner/UpdateNumericDataSample.java:2]:
```suggestion
 * Copyright 2020 Google LLC
```

**Review by chingor13 — APPROVED** [review]
Nits on the license header but otherwise LGTM

**Pull request merged**